### PR TITLE
update desyn core chain strategy pool

### PR DESCRIPTION
**Pull request by santinoYoung**
---
##### pool Addresses
0x5e69d826D3663094321E2cf3C387b7F9Dd7b44Bb

##### Chain:
core

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       123.59 M
    mode                      40.63 M
    zklink                    272.36 k
    core                      23
    
    total                    164.50 M

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm2: 
        
 
  
    
    
    TypeError: params.calls.map is not a function
    Truncated error stack:
        at multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/abi2.js:50:33)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm2/index.js:4:30)
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:165:29
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:160:44
        at Object.<anonymous> (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:258:3)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       123.67 M
    mode                      40.63 M
    zklink                    272.36 k
    core                      23
    
    total                    164.57 M

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm2: 
        
 
  
    
    
    TypeError: params.calls.map is not a function
    Truncated error stack:
        at multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/abi2.js:50:33)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm2/index.js:6:30)
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:165:29
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:160:44
        at Object.<anonymous> (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:258:3)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       123.81 M
    mode                      40.63 M
    zklink                    272.36 k
    core                      23
    
    total                    164.71 M

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm2 exports TVL: 
        
 
 
    core                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       124.64 M
    mode                      40.52 M
    zklink                    21.39 M
    core                      23
    
    total                    186.55 M

**Comment by llamatester** [comment]
The adapter at projects/desyn exports TVL: 
        
 
 
    ethereum                  4.56 M
    arbitrum                  245
    
    total                    4.56 M

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm2 exports TVL: 
        
 
 
    core                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    2.40 M
    
    total                    2.40 M

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       124.63 M
    mode                      40.91 M
    zklink                    21.55 M
    core                      23
    
    total                    187.09 M

**Comment by llamatester** [comment]
The adapter at projects/desyn exports TVL: 
        
 
 
    ethereum                  4.56 M
    arbitrum                  246
    
    total                    4.56 M

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm2 exports TVL: 
        
 
 
    core                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'map')
    Truncated error stack:
        at multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/abi2.js:50:33)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:32:43
        at Array.forEach (<anonymous>)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:31:14)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 2)

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:232:11
    Error saving cache: Error: ENOENT: no such file or directory, open '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
        at Object.openSync (node:fs:596:3)
        at Object.writeFileSync (node:fs:2322:35)
        at process.saveCache (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/cache.js:152:17)
        at process.emit (node:events:529:35)
        at process.exit (node:internal/process/per_thread:191:15)
        at process.handleError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/utils/handleError.js:18:11)
        at process.emit (node:events:529:35)
        at emit (node:internal/process/promises:149:20)
        at processPromiseRejections (node:internal/process/promises:283:27)
        at process.processTicksAndRejections (node:internal/process/task_queues:96:32) {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
    }

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:232:11
    Error saving cache: Error: ENOENT: no such file or directory, open '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
        at Object.openSync (node:fs:596:3)
        at Object.writeFileSync (node:fs:2322:35)
        at process.saveCache (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/cache.js:152:17)
        at process.emit (node:events:529:35)
        at process.exit (node:internal/process/per_thread:191:15)
        at process.handleError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/utils/handleError.js:18:11)
        at process.emit (node:events:529:35)
        at emit (node:internal/process/promises:149:20)
        at processPromiseRejections (node:internal/process/promises:283:27)
        at process.processTicksAndRejections (node:internal/process/task_queues:96:32) {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
    }

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:232:11
    Error saving cache: Error: ENOENT: no such file or directory, open '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
        at Object.openSync (node:fs:596:3)
        at Object.writeFileSync (node:fs:2322:35)
        at process.saveCache (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/cache.js:152:17)
        at process.emit (node:events:529:35)
        at process.exit (node:internal/process/per_thread:191:15)
        at process.handleError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/utils/handleError.js:18:11)
        at process.emit (node:events:529:35)
        at emit (node:internal/process/promises:149:20)
        at processPromiseRejections (node:internal/process/promises:283:27)
        at process.processTicksAndRejections (node:internal/process/task_queues:96:32) {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
    }

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:232:11
    Error saving cache: Error: ENOENT: no such file or directory, open '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
        at Object.openSync (node:fs:596:3)
        at Object.writeFileSync (node:fs:2322:35)
        at process.saveCache (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/cache.js:152:17)
        at process.emit (node:events:529:35)
        at process.exit (node:internal/process/per_thread:191:15)
        at process.handleError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/utils/handleError.js:18:11)
        at process.emit (node:events:529:35)
        at emit (node:internal/process/promises:149:20)
        at processPromiseRejections (node:internal/process/promises:283:27)
        at process.processTicksAndRejections (node:internal/process/task_queues:96:32) {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
    }

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    TypeError: tokens.forEach is not a function
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:31:14)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 2)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall failed! 
     [chain: btr] [fail count: 1] [abi: address:getBalance]  
        target: 0xfe9f969faf8Ad72a83b761138bF25dE87eFF9DD2

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: Cannot read properties of undefined (reading 'selector')

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: Cannot read properties of undefined (reading 'selector')

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: Cannot read properties of undefined (reading 'selector')

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall failed! 
     [chain: mode] [fail count: 1] [abi: address:getBalance]  
        target: 0xf0F161fDA2712DB8b566946122a5af183995e2eD

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn exports TVL: 
        
 
 
    arbitrum                  0
    ethereum                  0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm2 exports TVL: 
        
 
 
    core                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall failed! 
     [chain: mode] [fail count: 1] [abi: address:getBalance]  
        target: 0xf0F161fDA2712DB8b566946122a5af183995e2eD

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall failed! 
     [chain: core] [fail count: 1] [abi: address:getBalance]  
        target: 0x8034aB88C3512246Bf7894f57C834DdDBd1De01F

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall failed! 
     [chain: btr] [fail count: 1] [abi: address:getBalance]  
        target: 0xfe9f969faf8Ad72a83b761138bF25dE87eFF9DD2

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall failed! 
     [chain: mode] [fail count: 1] [abi: address:getBalance]  
        target: 0xf0F161fDA2712DB8b566946122a5af183995e2eD

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    ReferenceError: any is not defined
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:29:103)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 4)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'map')
    Truncated error stack:
        at multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/abi2.js:50:33)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:29:32)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 2)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading '2')
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:29:60
        at Array.map (<anonymous>)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:29:24)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 2)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall failed! 
     [chain: mode] [fail count: 1] [abi: address:getBalance]  
        target: 0xf0F161fDA2712DB8b566946122a5af183995e2eD

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: missing arguemnt: types/values length mismatch (count=0, expectedCount=1, code=MISSING_ARGUMENT, version=6.13.1)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: missing arguemnt: types/values length mismatch (count=0, expectedCount=1, code=MISSING_ARGUMENT, version=6.13.1)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Invalid target: 0xf0F161fDA2712DB8b566946122a5af183995e2eD
    Truncated error stack:
        at Object.multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/index.js:180:15)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:36:44
        at Array.forEach (<anonymous>)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:35:14)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: missing arguemnt: types/values length mismatch (count=0, expectedCount=1, code=MISSING_ARGUMENT, version=6.13.1)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: missing arguemnt: types/values length mismatch (count=0, expectedCount=1, code=MISSING_ARGUMENT, version=6.13.1)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'map')
    Truncated error stack:
        at multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/abi2.js:50:33)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:36:44
        at Array.forEach (<anonymous>)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:35:14)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 4)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    TypeError: Cannot read properties of undefined (reading 'map')
    Truncated error stack:
        at multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/abi2.js:50:33)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:35:42)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 1)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: missing arguemnt: types/values length mismatch (count=0, expectedCount=1, code=MISSING_ARGUMENT, version=6.13.1)

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: missing arguemnt: types/values length mismatch (count=0, expectedCount=1, code=MISSING_ARGUMENT, version=6.13.1)

File: projects/desyn-farm2/index.js
Comment (g1nt0ki) [inline projects/desyn-farm2/index.js:1]: removing this file, it looks like duplicate of 'desyn-farm1' file

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       231.05 M
    core                      64.26 M
    mode                      41.22 M
    zklink                    21.67 M
    
    total                    358.20 M

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    ReferenceError: tokens is not defined
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:35:81)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 5)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    ReferenceError: tokens is not defined
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:35:81)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 2)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by g1nt0ki** [comment]
@santinoYoung can you stop pushing commits

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    0
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/desyn-farm1: 
        
 
  
    
    
    Error: Multicall is missing a target
    Truncated error stack:
        at Object.multiCall (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/index.js:199:19)
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/desyn-farm1/index.js:35:37)
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:177:9
        at async Promise.all (index 0)
        at async Promise.all (index 3)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:220:3

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    2.40 M
    
    total                    2.40 M

**Comment by llamatester** [comment]
The adapter at projects/desyn exports TVL: 
        
 
 
    ethereum                  4.57 M
    arbitrum                  247
    
    total                    4.57 M

**Pull request merged**